### PR TITLE
Add output key to logs

### PR DESCRIPTION
**Pull request by vine77**
Ticket: https://tickets.puppetlabs.com/browse/PN-176

### CODE REVIEW

File: pkg/cmd/outputs.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ func NewSetOutputCommand() *cobra.Command {
 				return err
 			}
 
-			fmt.Fprintln(cmd.OutOrStdout(), "Successfully set output.")
+			fmt.Fprintln(cmd.OutOrStdout(), `Successfully set output for "`+key+`".`)
```
Comment (impl) [inline pkg/cmd/outputs.go]: You can get quotes with escaping automatically in Go using `strconv.Quote(string)` or using the `%q` modifier to `Printf`-style functions (you'd have to change this to `fmt.Fprintf` and put a newline at the end). Other than that, this looks good, thank you!

**Comment by impl** [comment]
Also, it would be great to update the `CHANGELOG` file with some details about this change!

File: pkg/cmd/outputs.go (hunk 1 above)
Comment (kyleterry) [inline pkg/cmd/outputs.go]: I would use a format string here. I find concatenation to be a little clunky.

```suggestion
			if _, err := fmt.Fprintf(cmd.OutOrStdout(), "Successfully set output for %s.\n", key); err != nil {
			    return err
			}
```

 https://golang.org/pkg/fmt/#Fprintf

**Comment by vine77** [comment]
Thanks for the help! I added the error check and usage of `Fprintf` per your suggestion @kyleterry and a note in CHANGELOG.md.

**Pull request merged**